web/src/SidebarResources.tsx

```diff
@@ -108,20 +122,31 @@ const SidebarGroupDetails = styled(AccordionDetails)`
 
 const GROUP_INFO_TOOLTIP_ID = "sidebar-groups-info"
 
-export function SidebarListSection(
-  props: PropsWithChildren<{ name: string }>
-): JSX.Element {
+export function SidebarListSection(props: SidebarSectionProps): JSX.Element {
   return (
-    <div>
-      <SidebarListSectionName>{props.name}</SidebarListSectionName>
-      <SidebarListSectionItems>{props.children}</SidebarListSectionItems>
-    </div>
+    <>
+      {props.sectionName && (
+        <SidebarListSectionName>{props.sectionName}</SidebarListSectionName>
+      )}
+      <SidebarListSectionItems {...props} />
+    </>
   )
 }
 
-function SidebarItemsView(props: SidebarProps & { groupView?: boolean }) {
+function SidebarListSectionItems(props: SidebarSectionProps) {
```

Comment:
tbh, "Items" feels like a weird name for a standalone react component

Reply:
haha yeah, naming all these components in this file is challenging! i opted for `items` to better describe the `li` in `ul`, but i may tweak the names a bit more in the pr following this one.